Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
+ADD ca-certificates.crt /etc/ssl/certs/
```

Comment:
o.O look at that:  

If <src> is a URL and <dest> does not end with a trailing slash, then a file is downloaded from the URL and copied to <dest>.

If <src> is a URL and <dest> does end with a trailing slash, then the filename is inferred from the URL and the file is downloaded to <dest>/<filename>. For instance, ADD http://example.com/foobar / would create the file /foobar. The URL must have a nontrivial path so that an appropriate filename can be discovered in this case (http://example.com will not work).

Will fix it to use ADD + https url